expo-app/app/components/CampusMap/CampusMapNavTab.tsx

```diff
@@ -30,14 +30,11 @@ const NavTab: React.FC<NavTabProps> = ({
 }) => {
   const [activeTab, setActiveTab] = useState<string | null>(null);
 
-  const NAV_ITEMS = selectedBuilding
-    ? [
-        { label: "Back", icon: "arrow-left", action: onBackPress },
-        { label: "Info", icon: "info-circle", action: onInfoPress },
-        { label: "Directions", icon: "route", action: onDirectionsPress },
-      ]
-    : [
-        { label: "Navigate", icon: "compass", action: onNavigatePress },
+  const getNAV_ITEMS = () => {
+
+    // No Destination
+    if (!destination) return [
+        { label: "Search", icon: "search", action: onSearchPress },
         {
           label: campus === "SGW" ? "SGW" : "LOY",
```

Comment:
is there a reason we don't just set label: campus
```suggestion
          label: campus,
```